### DISCUSSION TITLE
Sign in problem

### DISCUSSION THREAD

**Discussion by zubairahman** [community/community#42763] [New to GitHub] [answered]
I can't login to my github account after I put my email and password the two factor authentication show me this "We tried sending an SMS to your configured number, but there was an unexpected error sending the SMS. Please contact support if you continue to have problems." i don't have any access to login to my account please help me out

**Comment by SaifMahmud366** [comment]
1. please check you spam folder on email,
2. check if **Github is blocked on email or not**.
3. contact support

**Reply by zubairahman** [reply]
no email in spam....
actually i don't know how can i get best solution from  support.....

**Reply by zubairahman** [reply]
Github contract us virtual assistant not helping much..

**Comment by airtower-luna** [comment, marked as answer]
This is the support form for account problems: https://support.github.com/contact/account

Or you can go to account recovery: https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials
It should be painless as long as you have your recovery codes. Of course that won't solve the SMS problem (unless that was just a temporary glitch), but it should let you switch the 2FA method if you want. TOTP doesn't need any kind of connection once set up (that's why it's safer), but you need to make sure not to lose the secret. Either way make sure you don't use up all your recovery codes before generating new ones.
